packages/controller/src/resources/apis/systemlogs.ts

```diff
@@ -27,6 +27,75 @@ import { Tenant } from "@opstrace/tenants";
 import { KubeConfig } from "@kubernetes/client-node";
 import { getTenantNamespace, getControllerConfig } from "../../helpers";
 import { DockerImages, getImagePullSecrets } from "@opstrace/controller-config";
+import axios, { AxiosResponse } from "axios";
```

Comment:
let's use `httpcl` based on `got` when we are in the nodejs runtime as we are in the controller.

Much better timeout control and error handling.

Let's use axios only in the browser if we have to. (`ky`) would be my preferred option.

Reply:
> let's use httpcl based on got when we are in the nodejs runtime as we are in the controller.

But the controller is not using `got` it's using `axios`. Here's the prior art:  

https://github.com/opstrace/opstrace/blob/main/packages/controller/src/tasks/cortexRules.ts#L63

https://github.com/opstrace/opstrace/blob/main/packages/controller/src/tasks/syncTenants.ts#L70

What's the history on that then?

Reply:
Ok, I ended up moving to httpcl anyway. :shrug: